modules/gdnative/pluginscript/pluginscript_language.h

```diff
@@ -78,7 +78,7 @@ class PluginScriptLanguage : public ScriptLanguage {
 	virtual Script *create_script() const;
 	virtual bool has_named_classes() const;
 	virtual bool supports_builtin_mode() const;
-	virtual bool can_inherit_from_file() { return true; }
+	virtual bool can_inherit_from_file();
```

Comment:
Should this be `const` like the other ones?

Reply:
Totally agree, however the parent method is not const:
https://github.com/godotengine/godot/blob/d9f0477dad0aeb92d16da8d8bff37bece0f9fedd/core/script_language.h#L313

I can provide another PR to fix this if you want ;-)